arrow/translate.py

```diff
@@ -0,0 +1,67 @@
+import base64
+
+
+async def translate(reader, options=None):
+    if options is None:
+        options = {}
+    t = Translator(options)
+    return t.translate(reader)
+
+
+class Translator:
+
+    def __init__(self, options=None):
+        if options is None:
+            options = {}
+        defaults = {'b64-decode-enums': False, 'prefix-object-ids': False}
+        self.options = {**defaults, **options}
+
+    def translate(self, reader):
+        enums = _list_enums(reader.writer_schema)
+        results = []
+        for record in reader:
+            if record['name'] != 'metadata':
+                results.append(self._translate_record(record, enums))
+        print(len(results))
+        return results
+
+    def _translate_record(self, record, enums):
+        entity_type = record['name']
```

Comment:
I can check fastavro docs, but I suspect `None` isn't possible here... and if it happens it would be the type of unexpected failure that we couldn't do anything reasonable with anyway.

Reply:
I'm nearly certain that a fastavro `reader` will not return a `None` record. Since this function is intended to be internal to this class, I don't think the guard is needed.